requirments_docs.txt

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+furo>=2021.11.23
+sphinx==4.2.0
```

Comment:
in `requirment.txt` we add blank around the binary operator, which means:
```
furo >= 2021.11.23
sphinx == 4.2.0
```
I think we should keep the style consistent.